packages/evm/test/eof/eofContainer.spec.ts

```diff
@@ -0,0 +1,188 @@
+import { toBuffer } from '@ethereumjs/util'
```

Comment:
@ScottyPoi cool, thanks for addressing the comments so quickly, short nit: this would then need a renaming as well. 🙂